### ISSUE TITLE
Update README to include how to run GrimoireLab

### ISSUE THREAD

**Issue by jsmanrique** [chaoss/grimoirelab#220]
Potentially related with:
* chaoss/grimoirelab#135
* chaoss/grimoirelab-tutorial#93

I am missing information about how to run GrimoireLab in its project README file. At least a basic structure like:

# Getting started

Of course, first, you need git and to clone this repository.

You need to modify at least 2 configuration files to get ready:
* `projects.json` to describe the projects and data sources you wanna analyze. Read more about this file in (link to further information)
* `setup.cfg` to configure some procedures. By now, just don't forget to uncomment those sections related to the data sources your listed in the projects.json (this shouldn't be needed if chaoss/grimoirelab-sirmordred#321 gets solved). Read more about this file in (link to further information)
* `credentials.cfg` to list those tokens needed by some data sources. Read more about this file in (link to further information)

Once you are ready, there are several ways to start playing with GrimoireLab toolkit:

## To play with GrimoireLab using pip packages
Requirements: git, Elasticsearch, Kibiter (or Kibana), MariaDB, Redis, ...

Run:
```
$ pip install grimoirelab
```
Run:
```
$ mordred -c setup.cfg credentials.cfg
```

Link to further reading about GrimoireLab python packages ...

## To play with GrimoireLab using a single Docker image

Requirements: git, Docker Engine

Run:
```
$ docker run -p 127.0.0.1:9200:9200 -p 127.0.0.1:5601:5601 \
    -v $(pwd)/logs:/logs \
    -v $(pwd)/credentials.cfg:/override.cfg \
    -v $(pwd)/projects.json:/projects.json \
    -t grimoirelab/full
```

Link to further reading about GrimoireLab single Docker image ...

## To play with GrimoireLab using Docker Compose

Requirements: git, Docker Engine, Docker Compose

Modify settings files and run:
```
$ docker-compose up -d
```

Link to further reading about GrimoireLab using docker-compose ...

**Comment by sduenas** [comment]
@jsmanrique can you check the current version of the README file. Maybe it contains all you described here. If doesn't not the case, please create a PR adding the text you wanted to include.

**Comment by jsmanrique** [comment]
I think this issue can be closed thanks to #221

**Comment by sduenas** [comment]
Thanks!
